docs/source/tutorials/configuration.ipynb

```diff
@@ -1,20 +1,452 @@
 {
```

Comment:
<p>Some wording suggestions:</p><ul><li>All this information is provided to the pilot of a RP application via the <code>radical.pilot.PilotDescription</code> object -&gt; All this information is provided to the pilot of a RP application via the <code>radical.pilot.PilotDescription</code>  (pd) object</li><li>Among that object's attributes, <code>resource</code> is mandatory and requires an HPC platform ID as its value. -&gt; &nbsp;The resource is mandatory among that object's attributes and requires an HPC platform ID as its value.</li><li>Users can find these pre-defined IDs -&gt; Users can find these predefined IDs</li><li>offers a general-purpose set of configuration parameters -&gt; offers a set of general-purpose configuration parameters</li><li>Users can customize existing platform configuration files by overwriting existing key/value pairs -&gt; Users can customize platform configuration files by overwriting key/value pairs.</li><li>creating a configuration file -&gt; Creating a configuration file</li><li> that ID and specific set of configurations -&gt;  that ID and a specific set of configurations</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>